api/src/main/java/au/org/aodn/nrmn/restapi/service/SpreadSheetService.java

```diff
@@ -42,150 +38,47 @@
 
     @Autowired
     private S3IO s3client;
+    
+    public Validated<ErrorInput, List<StagedRow>> stageXlsxFile(MultipartFile file, Boolean withInvertSize) {
+        try {
+            InputStream inputStream = file.getInputStream();
```

Comment:
This needs to be closed in a finally to ensure its always closed otherwise we can un out of file system handles (or whatever they're called)

Reply:
Better yet use a try with resources block

Reply:
try (InputStream inputStream = file.getInputStream()) {
    ...
}